docs/theory/diffusion/diffusion.rst

```diff
@@ -0,0 +1,715 @@
+        "cve (filtered)": lambda list_of_tracks: [t.estimate_diffusion("cve") for tracks in list_of_tracks for t in lk.filter_tracks(tracks, min_length)],
+        "ols (filtered)": lambda list_of_tracks: [t.estimate_diffusion("ols") for tracks in list_of_tracks for t in lk.filter_tracks(tracks, min_length)],
+        "cve ensemble": lambda list_of_tracks: [t.ensemble_diffusion("cve") for t in list_of_tracks],
+        "ols ensemble": lambda list_of_tracks: [t.ensemble_diffusion("ols") for t in list_of_tracks],
+        "cve ensemble (filtered)": lambda list_of_tracks: [lk.filter_tracks(t, min_length).ensemble_diffusion("cve") for t in list_of_tracks],
+        "ols ensemble (filtered)": lambda list_of_tracks: [lk.filter_tracks(t, min_length).ensemble_diffusion("ols") for t in list_of_tracks],
+    }
+
+    snrs = 10**np.arange(-1, 1.1, 0.125)
+    results, variances = test_estimation_methods(simulate_tracks, snrs, methods=methods)
+
+    plt.figure(figsize=(12, 4))
+    plt.subplot(1, 2, 1)
+    for method in ("cve single", "ols single", "cve (filtered)", "ols (filtered)"):
+        plot_accuracy(snrs, results[method], variance=variances[method], label=method);
+    plt.ylim([-1, 3])
+
+    plt.subplot(1, 2, 2)
+    for method in ("cve ensemble", "ols ensemble", "cve ensemble (filtered)", "ols ensemble (filtered)"):
+        plot_accuracy(snrs, results[method], variance=variances[method], label=method);
+    plt.ylim([-1, 3])
+
+.. image:: figures/filtering_short_tracks.png
+
+The good news is that the ensemble estimates are pretty robust to filtering short tracks.
+This is expected since these estimates already incorporate weighting (assigning lower weight to short tracks).
+The bad news is that the variance for low SNR is still very high, while the method with the most precision at low SNR (OLS) is very biased.
+Unfortunately, what this means is that we either need to settle for ensemble estimates, or accept high variance results.
+
+Concluding remarks
+------------------
+When the SNR and the number of points per track is high, all methods presented here perform well.
+In this case, the recommendation is to use CVE, while carrying over the localization uncertainty from the ensemble estimate.
+Choosing this option will allow the study of diffusion constants for individual tracks with the largest precision.
+
+When the SNR is low, but per track estimates are required, restrict to long tracks if possible (refine the tracks and filter short tracks out) and use OLS.
+
+If the uncertainty in the diffusion estimates is still too large, but the expectation is that the diffusion constants for different tracks are the same, consider using ensemble averages and compare ensemble averages obtained with CVE and OLS.
+Ensemble estimates perform well down to relatively low SNR.
+Note that a extra requirement for OLS is that the diffusion constants are roughly equally long.
```

Comment:
what do you mean by "equally long"?

Reply:
Oh no, that sentence makes no sense 😅 

I meant:
``` 
Note that an extra requirement for OLS is that the diffusive tracks are roughly equally long.
````